_sass/_install.scss

```diff
@@ -0,0 +1,23 @@
+.install-nav {
+
+  .nav-link {
+    color: $yarn-blue-darkest !important;
+  }
+}
+
+.problem-links {
```

Comment:
Please adjust these colors inside of all guide blockquotes: https://github.com/yarnpkg/website/blob/master/_sass/_guides.scss#L21